src/rust/ide/view/graph-editor/src/component/node/output/area.rs

```diff
@@ -122,6 +121,7 @@ ensogl::define_endpoints! {
         set_hover                 (bool),
         set_expression            (node::Expression),
         set_expression_visibility (bool),
+        set_port_label_visibility (bool),
```

Comment:
What port labels have to do with types under the node? This change seems completely unrelated to the subject of this PR.